.github/workflows/go.yml

```diff
@@ -0,0 +1,51 @@
+name: Go
+
+on:
+  push:
+  pull_request:
+    # edtited - "title, body, or the base branch of the PR is modified"
+    # synchronize - "commit(s) pushed to the pull request"
+    types: [opened, synchronize, edited, reopened]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-16.04, windows-2019, macos-10.15 ]
+    name: Testing on on ${{ matrix.os }}
+    steps:
+
+      - name: Fix windows CRLF
+        run: git config --global core.autocrlf false
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      # We need to increase the page size because the tests run out of memory on github CI windows.
+      # Use the powershell script from this github action: https://github.com/al-cheb/configure-pagefile-action/blob/master/scripts/SetPageFileSize.ps1
+      # MIT License (MIT) Copyright (c) 2020 Maxim Lobanov and contributors
+      - name: Increase page size on windows
+        if: runner.os == 'Windows'
+        shell: powershell
+        run: powershell -command .\.github\workflows\SetPageFileSize.ps1
+
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
```

Comment:
Currently we are still at version 1.14
Upgrade to 1.15 should happen for everything at the same time.

Reply:
Done